### PR TITLE
Pluralize menu entry

### PR DESCRIPTION
**Pull request by AcelisWeaven**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

This PR pluralize the menu entry for the articles, by replacing "Article" with "Articles".

#### Why?

Most entries are already pluralized: "Snippets", "Users", ... the goal of this PR is to use the same convention on this entry.

By the way I don't speak german or dutch, so I'm not 100% sure of the pluralization here.

### CODE REVIEW

File: Resources/translations/sulu/backend.de.xlf [hunk 1]
```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1" resname="sulu_article.title">
                 <source>sulu_article.title</source>
-                <target>Artikel</target>
+                <target>Artikels</target>
```
Comment (trickreich) [inline Resources/translations/sulu/backend.de.xlf]: In German it is ` Artikel`

File: Resources/translations/sulu/backend.de.xlf [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
             </trans-unit>
             <trans-unit id="2" resname="sulu_article.list.title">
                 <source>sulu_article.list.title</source>
-                <target>Artikel</target>
+                <target>Artikels</target>
```
Comment (trickreich) [inline Resources/translations/sulu/backend.de.xlf]: In German it is ` Artikel`

File: Resources/translations/sulu/backend.nl.xlf [hunk 3]
```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1" resname="sulu_article.title">
                 <source>sulu_article.title</source>
-                <target>Artikel</target>
+                <target>Artikels</target>
```
Comment (trickreich) [inline Resources/translations/sulu/backend.nl.xlf]: I think in NL it is `Artikelen`

**Comment by trickreich** [comment]
@AcelisWeaven Thanks for the contribution! :+1: 
Can you fix the comments?
Then we can merge this.

**Comment by AcelisWeaven** [comment]
Thanks for the feedback @trickreich ! That should be better now.

**Pull request merged**